bwi_logging/src/bwi_logging/logger_node.py

```diff
@@ -69,13 +71,16 @@ def main(argv=None):
     rospy.loginfo('logs go here: ' + logdir.pwd())
     logdir.chdir()                      # change to that directory
 
+    # get the file prefix
+    prefix = rospy.get_param('~prefix', 'bwi')
```

Comment:
rosbag automatically adds an underscore after the specified prefix, so it is not needed